signer/src/bitcoin/utxo.rs

```diff
@@ -54,10 +54,11 @@ const DEFAULT_INCREMENTAL_RELAY_FEE_RATE: f64 =
     bitcoin::policy::DEFAULT_INCREMENTAL_RELAY_FEE as f64 / 1000.0;
 
 /// This constant represents the virtual size (in vBytes) of a BTC
-/// transaction that includes two inputs and one output. The inputs
-/// consist of the signers' input UTXO and a UTXO for a deposit request.
-/// The output is the signers' new UTXO.
-const SOLO_DEPOSIT_TX_VSIZE: f64 = 234.0;
+/// transaction that includes two inputs and one output. The inputs consist
+/// of the signers' input UTXO and a UTXO for a deposit request. The output
+/// is the signers' new UTXO. The deposit request is such that the sweep
+/// transaction has the largest size of solo deposit sweep transactions.
+const SOLO_DEPOSIT_TX_VSIZE: f64 = 267.0;
```

Comment:
sanity check and...personal curiosity: how is this value computed? Just asking to better understand the context

Reply:
We have a "test", `create_deposit_only_tx`, that creates a sweep transaction moving only one deposit and zero withdrawals that outputs the size of the sweep transaction after it has been signed. This gives us the maximum size of a sweep transaction with only one deposit. Adding additional deposits decreases the costs, and adding additional withdrawals used to decrease the cost as well. 🤔 But now maybe not, I need to check again.

Reply:
I just checked and costs decrease with an added withdrawal.